packages/executable/package.json

```diff
@@ -13,6 +13,7 @@
   "bin": "../server/bin.js",
   "pkg": {
     "assets": [
+      "../../node_modules/sqlite3/lib/binding/**/*",
```

Comment:
I don't think this is right @lilitkarapetyan 

When this runs, the `sqlite3` folder will only contain the lib for the current operating system.

![image](https://user-images.githubusercontent.com/1108513/173557005-b8f60fc7-2719-49f0-8cd1-ed9644c97875.png)

But, we want `yarn pkg-build` to create executables for all 3 operating systems.  So, I think we have to manually put the 3 compiled `.node` files into the `executable\sqlite3.nodes` folder.